portfolio/src/main/webapp/style.css

```diff
@@ -7,3 +7,8 @@
 #greeting-container {
   margin-top: 20px;
 }
+
```

Comment:
Optional stretch: Use this CSS to add some basic style, like a font style + size, etc.